### PR TITLE
Remove obsolete references to 'kubeadm alpha kubeconfig' command

### PR DESCRIPTION
**Pull request by benileo**
It looks like 'kubeadm alpha kubeconfig' was deprecated as of 1.21 and removed as of 1.22.
This PR fixes the references to that command in the creating a cluster with kubeadm documentation. 

Removed in 1.22
https://github.com/kubernetes/kubernetes/pull/101938

Deprecated in 1.21
https://github.com/kubernetes/kubernetes/pull/97583

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
@benileo Thanks for your contribution. It seems that this PR contains modifications to files in various languages (fr, id, ja, zh-cn). However, docs follows distinct process for each localization, and we generally avoid accepting pull requests that affect multiple languages at once. My suggestion is to focus on altering only the primary website content (i.e only English) with this PR initially. Once the English content has been merged, we can then create separate pull requests for the remaining languages.

**Comment by dipesh-rawat** [comment]
/hold

Hold till the PR addresses changes only related to a single language.

**Comment by benileo** [comment]
Sounds good @dipesh-rawat. I have backed out the non-english changes.

**Review by neolit123 — COMMENTED** [review]
LGTM on the change, but please squash the commits to one
/sig cluster-lifecycle

**Comment by dipesh-rawat** [comment]
Removing unnecessary labels.
/remove-language fr
/remove-language id
/remove-language ja
/remove-language zh

**Comment by dipesh-rawat** [comment]
/unhold

PR now addresses changes related to a single language.

**Comment by benileo** [comment]
Squash'd

Should I open up PRs to fix the other languages as well?

**Comment by atoato88** [comment]
I think it's better that you create the PRs on other languages once this PR is merged.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Review by stormqueen1990 — APPROVED** [review]

**Pull request merged**
